### PR TITLE
Fix all typos found by a spell checker

### PR DESCRIPTION
**Pull request by jirkadanek**

### CODE REVIEW

File: src/dotNet/ClientLib/ReconnectSender.cs [hunk 1]
```diff
@@ -5,7 +5,7 @@
 //
 //  THIS CODE IS PROVIDED *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
 //  EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY IMPLIED WARRANTIES OR
-//  CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE, MERCHANTABLITY OR
+//  CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE, MERCHANTABILITY OR
```
Comment (jirkadanek) [inline src/dotNet/ClientLib/ReconnectSender.cs:8]: Is this typo also in Chuck's example?

File: src/dotNet/ClientLib/ReconnectSender.cs [hunk 2]
```diff
@@ -14,9 +14,9 @@
 
 //
 // ReconnectSender 
-//  * Detects a failed AMQP connection and automatically reconnets it.
+//  * Detects a failed AMQP connection and automatically reconnects it.
 //  * Recovers from a peer failure by connecting to a list of AMQP brokers/peers.
-//  * Builds or rebuilds the AMPQ object hierarchy in response to protocol events.
+//  * Builds or rebuilds the AMQP object hierarchy in response to protocol events.
```
Comment (jirkadanek) [inline src/dotNet/ClientLib/ReconnectSender.cs:19]: Is this typo also in Chuck's example?

File: src/dotNet/ClientLib/ReconnectSender.cs [hunk 3]
```diff
@@ -14,9 +14,9 @@
 
 //
 // ReconnectSender 
-//  * Detects a failed AMQP connection and automatically reconnets it.
+//  * Detects a failed AMQP connection and automatically reconnects it.
```
Comment (jirkadanek) [inline src/dotNet/ClientLib/ReconnectSender.cs:17]: Is this typo also in Chuck's example?

File: src/dotNet/ClientLib/ReconnectSender.cs (hunk 1 above)
Comment (kornys) [reply src/dotNet/ClientLib/ReconnectSender.cs:8]: yep

File: src/dotNet/ClientLib/ReconnectSender.cs (hunk 2 above)
Comment (kornys) [reply src/dotNet/ClientLib/ReconnectSender.cs:19]: yep

File: src/dotNet/ClientLib/ReconnectSender.cs (hunk 3 above)
Comment (kornys) [reply src/dotNet/ClientLib/ReconnectSender.cs:17]: yep this example was copied from chuck

**Review by kornys — APPROVED** [review]
Thanks @jdanekrh for fixing typos 👍

**Comment by jirkadanek** [comment]
OK, will fix it in chucks code, before some customer reports it :( Typo in hello world...

**Comment by jirkadanek** [comment]
Done, Azure/amqpnetlite#298

**Pull request merged**